startup/gui/usd.py

```diff
@@ -54,3 +54,11 @@
 	"samples", "volume_samples", "resolution"
 ] ) :
 	Gaffer.Metadata.registerValue( GafferUSD.USDLight, f"parameters.arnold:{parameter}", "layout:index", 1000 + i )
+
+# Change Cycles ordering.
+for i, parameter in enumerate( [
+	"lightgroup",
+	"use_mis", "use_camera", "use_diffuse", "use_glossy", "use_transmission", "use_scatter", "use_caustics",
+	"spread", "map_resolution", "max_bounces"
+] ) :
+	Gaffer.Metadata.registerValue( GafferUSD.USDLight, f"parameters.cycles:{parameter}", "layout:index", 2000 + i )
```

Comment:
I think it'd be worth respecting `GAFFERCYCLES_HIDE_UI` and hiding these plugs if the user has decided to opt out of Cycles in the UI. I think we could do this with a visibility activator doing the usual Cycles environment variable checks along the lines of:

```
Gaffer.Metadata.registerValue( GafferUSD.USDLight, "parameters", "layout:activator:cyclesUIEnabled", lambda x : os.environ.get( "CYCLES_ROOT" ) and os.environ.get( "GAFFERCYCLES_HIDE_UI", "" ) != "1" )
Gaffer.Metadata.registerValue( GafferUSD.USDLight, "parameters.cycles:*", "layout:visibilityActivator", "cyclesUIEnabled" )
```